### DISCUSSION TITLE
Version 1.93.0 of VSCode breaks Prettier for Salesforce APEX

### DISCUSSION THREAD

**Discussion by bdaul-trail** [microsoft/vscode-discussions#1626] [Extension Development QnA]
Version 1.91.1 of Visual Studio Code looks like it was the last version to work with APEX formatting using Prettier.

A downgrade of VSCode to version 1.91.1 and Prettier does work as expected.

Will there be a fix for Prettier soon? 

Thanks

**Comment by gjsjohnmurray** [comment]
Why are you asking here? Please contact the extension developers. The Report Issue option in VS Code can help. The Extensions view can also link you to the repository.

**Comment by olguzzar** [comment]
Hi @bdaul-trail, as stated above, your question is about the Prettier extension specifically and should be filed as an issue in [their repo](https://github.com/prettier/prettier-vscode/issues) instead. This forum is for those who develop VS Code extensions, so I'm going to close this discussion. Thanks!
